pyteal/ast/abi/address.py

```diff
@@ -0,0 +1,33 @@
+from .array_static import StaticArray, StaticArrayTypeSpec
+from .uint import ByteTypeSpec
+from ..expr import Expr
+
+address_length = 32
```

Comment:
nit: probably uppercase this for it's being a constant, and probably export this?